docs/explanations/decisions/0003-codeowners.md

```diff
@@ -0,0 +1,17 @@
+# 3. Use CODEOWNERS Github feature to restrict approved reviewers
```

Comment:
I think it would be good to add something in this ADR about this likely being temporary. I'm generally quite opposed to closed shops like this and I would hope that once 80% of DAQ is on the list we can just drop the list with the idea that we have enough critical mass for it to not be necessary